src/components/SQForm/SQFormTextField.js

```diff
@@ -18,19 +18,45 @@ function SQFormTextField({
   startAdornment,
   endAdornment,
   type = 'text',
+  inputProps = {},
+  maxCharacters,
   muiFieldProps = {}
 }) {
   const {
     formikField: {field},
     fieldState: {isFieldError},
-    fieldHelpers: {handleBlur, handleChange, HelperTextComponent}
+    fieldHelpers: {
+      handleBlur,
+      handleChange: handleChangeHelper,
+      HelperTextComponent
+    }
   } = useForm({
     name,
     isRequired,
     onBlur,
     onChange
   });
 
+  const [valueLength, setValueLength] = React.useState(field.value.length || 0);
+
+  const handleChange = e => {
```

Comment:
https://selectquote.atlassian.net/wiki/spaces/SE/pages/796590204/Internal+Code+Standards#Naming

Worth a quick glance over the naming section. Try to avoid variables names that are a single letter.

Reply:
Ah, right. For some reason `event` is the only variable I do that to, but that should be an easy habit to break 👍